src/app/AuthenticationController.tsx

```diff
@@ -106,6 +108,11 @@ class AuthenticationController extends React.Component<AuthenticationControllerP
         .catch(error => {
           MessageCenter.add(API.getErrorMsg('Error fetching Grafana Info.', error), 'default', MessageType.WARNING);
         });
+      const getJaegerInfoPromise = API.getJaegerInfo()
+        .then(response => this.setJaegerInfo(response.data))
+        .catch(error => {
+          MessageCenter.add(API.getErrorMsg('Error fetching Jaeger Info.', error), 'default', MessageType.WARNING);
```

Comment:
Can we set message type to `MessageType.INFO` instead (and same for Grafana above) ?
So that the error is less intrusive